streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -682,7 +682,7 @@ public class StreamsConfig extends AbstractConfig {
 
     /** {@code processor.wrapper.class} */
     public static final String PROCESSOR_WRAPPER_CLASS_CONFIG = "processor.wrapper.class";
-    public static final String PROCESSOR_WRAPPER_CLASS_DOC = "A processor wrapper class or class name that implements the <code>org.apache.kafka.streams.state.ProcessorWrapper</code> interface. "
+    static final String PROCESSOR_WRAPPER_CLASS_DOC = "A processor wrapper class or class name that implements the <code>org.apache.kafka.streams.state.ProcessorWrapper</code> interface. "
```

Comment:
This is the root cause for the PR. Saw this by chance...

KIP-1112 correctly declares it as `private` -- we need to make it package-private though.,to allow `TopologyConfig` to use it. It should not be public.

As it seems we keep leaking these members (and it could happen again in the future), I added a test for it below (to avoid that we need to do another KIP to remove it again in the future... cf https://cwiki.apache.org/confluence/display/KAFKA/KIP-1085%3A+Fix+leaking+*_DOC+variables+in+StreamsConfig)

Took the liberty to fix this in 4.0 directly: https://github.com/apache/kafka/commit/562ecf4a83c2fa3577c45eba02328a1bfb751c2a